corehq/apps/callcenter/checks.py

```diff
@@ -60,7 +61,7 @@ def _compile_stats(name, ucr_count, es_count):
     total = ucr_count + es_count
 
     def _percent(count):
-        return 0 if total == 0 else int(100 * count / total)
+        return 0 if total == 0 else int(100 * count // total)
```

Comment:
Do these still need to be wrapped with `int()`?

Reply:
Probably not - my reasoning was that if ```count``` was ever a floating point value then ``` count // total``` would be an integer with floating point type (i.e. ```3.0```) but on closer inspection it looks like ```count``` is always an ```int```.  I'll simplify the code in a followup PR.

Reply:
https://github.com/dimagi/commcare-hq/pull/19441